LevelSpaceXWGUI.nlogo

```diff
@@ -539,31 +520,67 @@ to-report to-string [an-entity]
   report table:get last an-entity "to-string" 
 end
 
-
-
-
-to go
-  run-relationships
+to run-relationship [ rel-obj ]
+  let agent-obj table:get tasks (table:get rel-obj "agent-id")
+  let cmd-obj table:get tasks (table:get rel-obj "command-id")
+  let cmd-model table:get cmd-obj "model"
+  let cmd-args map last table:get rel-obj "command-arg-id-tuples"
+  
+  let agent-type table:get agent-obj "type"
+  let agent-model table:get agent-obj "model"
+  let agent-args map last table:get rel-obj "agent-arg-id-tuples"
+  let agent-arg-vals eval-args "x" "" agent-args
+  
+  (cf:match agent-type
+    cf:case [ ? = "observer" ] [
```

Comment:
Could this be `cf:=`? I'm not sure how the `cf` extension works, but I see `cf:=` on line 547.